### PR TITLE
HACK week: TypeScript migration: client/card-readers/setings

### PR DESCRIPTION
**Pull request by mordeth**
Fixes #

#### Changes proposed in this Pull Request

TypeScript migration for client/disputes/evidence

#### Testing instructions

Tests should pass without errors.

-------------------

- [x] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: client/card-readers/settings/index.tsx
Comment (mgascam) [inline client/card-readers/settings/index.tsx:23]: Is this intentional?

**Review by mgascam — CHANGES_REQUESTED** [review]
Please make sure that we are not changing the `isBrandingEnabled` flag by mistake

File: client/card-readers/settings/index.tsx
Comment (mordeth) [reply client/card-readers/settings/index.tsx]: Sorry, it is a mistake. I've changed it for testing purposes and forgot to set it back to `false`. Sorted.

File: client/card-readers/settings/file-upload.tsx
Comment (kalessil) [inline client/card-readers/settings/file-upload.tsx:16]: Can we move the interface in this file, please?

File: client/card-readers/settings/file-upload.tsx
Comment (kalessil) [inline client/card-readers/settings/file-upload.tsx:58]: Hm, this doesn't look right.

File: client/types/disputes.d.ts
Comment (kalessil) [inline client/types/disputes.d.ts:100]: Could we also re-specify the type here?

**Review by kalessil — CHANGES_REQUESTED** [review]
Looks great! I left some comments but will wait until Miguels' PR is merged and this one is updated for a better review.

File: client/card-readers/settings/file-upload.tsx
Comment (mgascam) [inline client/card-readers/settings/file-upload.tsx:40]: Can we remove `: CardReaderFileUploadProps`? Since we are already defining the type of props in `const BrandingFileUpload: React.FunctionComponent< CardReaderFileUploadProps >` I was wondering if TS compiler needs it?

File: client/card-readers/settings/file-upload.tsx
Comment (mgascam) [inline client/card-readers/settings/file-upload.tsx:88]: Would be `unknown` a better fit in this case?

**Review by mgascam — APPROVED** [review]
LGTM! I just left a couple of comments for your consideration, feel free to skip them

**Review by kalessil — APPROVED** [review]
LGTM! I didn't test the changes though.

**Pull request merged**
